notebooks/beginner.ipynb

```diff
@@ -614,19 +584,503 @@
    "source": [
     "k.expand.to_s"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## SymEngine::RealDouble\n",
+    "\n",
+    "SymEngine::RealDouble can be constructed by converting any ruby Float into SymEngine"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "1.2"
+      ],
+      "text/plain": [
+       "#<SymEngine::RealDouble(1.2)>"
+      ]
```

Comment:
This output is because of this code, https://github.com/symengine/symengine.rb/blob/master/lib/symengine/iruby.rb

This was done by @abinashmeher999 before `inspect` method showed the string representation, so that in iruby notebooks the object would print the contents instead of the memory address.
This should be changed though.